### PR TITLE
fix: [CDS-72881]: Removes FF CDS_OrgAccountLevelServiceEnvEnvGroup and updates dependent test cases

### PR DESCRIPTION
**Pull request by samarth-sharma31**
### Summary

This PR removes the FF for org/acc level service and environment in order to GA it and updates the service and env selection logic for dependent test cases (jest and cypress) across modules

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/10-common/components/ResourceCardList/ResourceCardList.tsx [hunk 1]
```diff
@@ -154,15 +146,15 @@ const ResourceCardList: React.FC<ResourceCardListProps> = ({ items }) => {
       icon: 'services',
       route: routes.toServices({ accountId, orgIdentifier }),
       colorClass: css.connectors,
-      hidden: !CDS_OrgAccountLevelServiceEnvEnvGroup,
+      hidden: false,
```
Comment (pankajsingh-harness) [inline src/modules/10-common/components/ResourceCardList/ResourceCardList.tsx:149]: NIT: We can remove the hidden prop from here as it only filters out the values.

Comment (pankajsingh-harness) [reply src/modules/10-common/components/ResourceCardList/ResourceCardList.tsx:149]: In the same file(Line: 269). 
```
options
          .filter(option => !option.hidden)
          .map(option => (
```

**Review by pankajsingh-harness — APPROVED** [review]

File: src/modules/10-common/components/ResourceCardList/ResourceCardList.tsx (hunk 1 above)
Comment (samarth-sharma31) [reply src/modules/10-common/components/ResourceCardList/ResourceCardList.tsx:149]: Sure @pankajsingh-harness , i'll raise a follow up PR for this change to reduce the scope of testing.

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -331,13 +325,18 @@ Cypress.Commands.add('addNewMonitoredServiceWithServiceAndEnv', () => {
   cy.contains('span', 'Environment is required').should('be.visible')
   cy.contains('span', 'Monitored Service Name is required').should('be.visible')
 
-  cy.get('input[name="service"]').click()
-  cy.contains('p', 'Service 101').click({ force: true })
+  // Select Service
+  cy.get('[data-testid="cr-field-serviceRef"]').click()
```
Comment (chetanvekariya-harness) [inline cypress/support/commands.ts:329]: We can use `cy.findByTestId` here.

File: cypress/support/commands.ts [hunk 3]
```diff
@@ -331,13 +325,18 @@ Cypress.Commands.add('addNewMonitoredServiceWithServiceAndEnv', () => {
...
+  cy.contains('p', 'DockerServicetest').click({ force: true })
+  cy.contains('span', 'Apply Selected').click({ force: true })
 
   cy.contains('span', 'Service is required').should('not.exist')
 
-  cy.get('input[name="environment"]').click()
-  cy.contains('p', 'Dev').click({ force: true })
+  // Select Env
+  cy.get('[data-testid="cr-field-environmentRef"]').click()
```
Comment (chetanvekariya-harness) [inline cypress/support/commands.ts:336]: Same as above

File: cypress/support/commands.ts [hunk 4]
```diff
@@ -569,20 +568,20 @@ Cypress.Commands.add('verifyStepSelectStrategyAndVerifyStep', () => {
 })
 
 Cypress.Commands.add('setServiceEnvRuntime', () => {
-  cy.get('div[data-testid="service"] span[data-icon="fixed-input"]').should('be.visible').click()
+  cy.get('div[class*="FormMultiTypeServiceField"] span[data-icon="fixed-input"]').should('be.visible').click()
```
Comment (chetanvekariya-harness) [inline cypress/support/commands.ts:571]: Why this change? Can't we use `cy.findByTestId` here? Same question for all similar changes in this file.

File: src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/__test__/AddEditServiceOverride.test.tsx [hunk 5]
```diff
@@ -49,59 +50,47 @@ describe('Add Edit Service Override Test', () => {
...
           selectedService={null}
           isReadonly={false}
-          services={mockServicesListForOverride.data.content}
+          services={[]}
         />
       </TestWrapper>
     )
 
-    const textboxes = screen.getAllByRole('textbox')
+    await userEvent.click(container.querySelector('[data-testid="cr-field-serviceRef"]') as HTMLElement)
```
Comment (chetanvekariya-harness) [inline src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/__test__/AddEditServiceOverride.test.tsx:58]: `findByTestId` can be used here.

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/DeployServiceEntityWidget.MultiService.test.tsx [hunk 6]
```diff
@@ -66,17 +75,17 @@ describe('DeployServiceEntityWidget - multi services tests', () => {
 
     await userEvent.click(toggle)
 
-    const multi = await findByTestId('multi-select-dropdown-button')
-
-    await userEvent.click(multi)
+    await userEvent.click(container.querySelector('[class="StyledProps--font StyledProps--main pointer"]')!)
```
Comment (chetanvekariya-harness) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/DeployServiceEntityWidget.MultiService.test.tsx:78]: Why this change?

**Review by chetanvekariya-harness — APPROVED** [review]

File: cypress/support/commands.ts (hunk 4 above)
Comment (samarth-sharma31) [reply cypress/support/commands.ts:571]: This component does not contain a test id

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/DeployServiceEntityWidget.MultiService.test.tsx (hunk 6 above)
Comment (samarth-sharma31) [reply src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/DeployServiceEntityWidget.MultiService.test.tsx:78]: These test cases were written for the old service env components which need to be queried in a certain way but then never got updated for org acc level service and env component (which was behind FF, globally enabled though).
So now since we are cleaning up the FF,  those components need to be queried in this way as the don't contain the identifiers like test id and dropdown buttons

File: cypress/support/commands.ts (hunk 2 above)
Comment (samarth-sharma31) [reply cypress/support/commands.ts]: done

File: src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/__test__/AddEditServiceOverride.test.tsx (hunk 5 above)
Comment (samarth-sharma31) [reply src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/__test__/AddEditServiceOverride.test.tsx]: done

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by pankajsingh-harness — APPROVED** [review]

**Review by sunnykesh-harness — APPROVED** [review]

**Review by deepesh-ui — APPROVED** [review]
LGTM 👍
